crates/basilica-sdk-python/examples/start_rental.py

```
@@ -0,0 +1,84 @@
+    
+    while time.time() - start_time < max_wait:
+        status = client.get_rental(rental_id)
+        state = status.get("status", {}).get("state", "Unknown")
+        
+        if state == "Active":
+            print("Rental is now active!")
+            
+            # Display SSH access info
+            if "ssh_access" in status:
+                ssh = status["ssh_access"]
+                print(f"\nSSH Access:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)